doc/content/examples/example3.md

```diff
@@ -87,4 +87,36 @@ DEEPSOIL and presented in
 Further details regarding to application of MASTODON to simulate seismic site response and soil
 structure interaction analysis can be found in [!citet](baltaji2017nonlinear).
 
+## Implementation using Automatic Differentiation
+
+This example can also be run using Automatic Differentiation (AD). Automatic Differentiation
```

Comment:
Please use the following text. 

This example can also be run using Automatic Differentiation (AD). Automatic Differentiation enables the computation of a numerical Jacobian that is almost identical to the theoretical Jacobian of a kernel. This results in much quicker convergence, even for highly nonlinear problems. Therefore, `solve_type=NEWTON` can be used in the executioner, which uses the full Jacobian unlike `solve_type=PJFNK`, which is recommended when only an approximate estimate of the Jacobian is available. In MASTODON, the iSoil material model has been converted to AD and the usage of the AD version of iSoil is demonstrated below. Further information on the AD version of the iSoil material models can be found in [here](source/materials/ADComputeISoilStress.md).